features/old/finance/invoices/multitenant.feature

```diff
@@ -1,3 +1,4 @@
+@wip
```

Comment:
Let's not forget to revert this. Meanwhile, we may want to add "WIP" to the description of the PR.

Reply:
good point! 

how about we revert this particular change (i think this was only done to see just how fast the build would be if this alone didn't take ~3mins or so...) and create a separate issue to track actually _fixing_ this?

Reply:
I'll revert this until we get coverage report. Then we should easily see if not running this test decreases test coverage or not and is covered by some other tests.

Reply:
@mikz i think we should get this merged first and then i can open a separate PR with just the coverage, right?

or would you prefer those commits here?

Reply:
Definitely a separace PR.